dashboard/app/assets/stylesheets/application.scss

```diff
@@ -290,6 +290,9 @@ img.video_thumbnail {
       text-decoration: none;
       background-color: $orange;
     }
+    &:active {
+      border: 2px solid $white !important;
```

Comment:
This is marked `!important` because it's overriding a piece of styling that is also marked `!important`:
https://github.com/code-dot-org/code-dot-org/blob/0767f38508cf677a17f65f23686597e6887f18a3/dashboard/app/assets/stylesheets/application.scss#L3325